src/Miki/MikiBotApp.cs

```diff
@@ -291,7 +291,7 @@ public override async Task ConfigureAsync(ServiceCollection serviceCollection)
 
             serviceCollection.AddSingleton<RedditService>();
             serviceCollection.AddSingleton<AchievementCollection>();
-            serviceCollection.AddScoped<AchievementService>();
+            serviceCollection.AddSingleton<AchievementService>();
```

Comment:
Let me know if changing this to singleton has ramifications. Reason I changed this is because there is an observable in achievementservice for unlocking achievements.